kits/agent-failure-investigator/index.html

```
@@ -0,0 +1,164 @@
+      <span class="fw-badge">✓ LangGraph</span>
+      <span class="fw-badge">✓ OpenAI Agents</span>
+      <span class="fw-badge">✓ CrewAI</span>
+      <span class="fw-badge">✓ AutoGen</span>
+      <span class="fw-badge">✓ Lamatic</span>
+      <span class="fw-badge">✓ Native</span>
+    </div>
+
+    <div class="section-title">Open a case</div>
+    <div id="cases"></div>
+    <div class="picker-actions">
+      <button id="run-btn" class="primary" disabled>Run investigation</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)